api/backend.proto

```diff
@@ -103,15 +103,13 @@ message AssignTicketsResponse {}
 // The service implementing the Backent API that is called to generate matches
 // and make assignments for Tickets.
 service Backend {
-  // GetMatches triggers execution of the specfied MatchFunction for each of the
+  // CreateProposal triggers execution of the specfied MatchFunction for each of the
   // specified MatchProfiles. Each MatchFunction execution returns a set of
-  // proposals which are then evaluated to generate results. GetMatches method
+  // proposals which are then evaluated to generate results. CreateProposal method
   // streams these results back to the caller.
-  // TODO: Should this be renamed to createProposal? It's not a "Get" if it's
-  // executing a MatchFunction.
-  rpc GetMatches(GetMatchesRequest) returns (stream GetMatchesResponse) {
+  rpc CreateProposal(CreateProposalRequest) returns (stream CreateProposalResponse) {
     option (google.api.http) = {
-      post: "/v1/backend/matches/createProposal"
+      post: "/v1/backend/proposal/create"
```

Comment:
/proposals/create

Proposal is the noun so it's supposed to be plural because it's represented as a collection.